src/http/common.cr

```diff
@@ -40,6 +40,10 @@ module HTTP
           body = UnknownLengthContent.new(io)
         end
 
+        if body.is_a?(Content) && expect_continue?(headers)
+          body.expect_continue
```

Comment:
Jus another nit bit: I'd prefer `expect_continue` to be an explicit setter for the ivar. `body.expect_continue = true` is more explicit about it's function.

Reply:
Good point! Update incoming.